jmh/src/main/scala/eu/neverblink/jelly/jmh/RdfIriParseBench.scala

```diff
@@ -53,25 +52,25 @@ class RdfIriParseBench:
   @OutputTimeUnit(TimeUnit.NANOSECONDS)
   @BenchmarkMode(Array(Mode.AverageTime))
   def currentImplementation(input: BenchInput): Unit =
-    val inputStream = input.getInputStream
+    val cis = input.getInputStream
     for i <- 0 until input.size do
       val iri = RdfIri.newInstance()
-      ProtoMessage.mergeDelimitedFrom(iri, inputStream)
+      ProtoMessage.mergeDelimitedFrom(iri, cis, ProtoMessage.DEFAULT_MAX_RECURSION_DEPTH)
 
-  @Benchmark
-  @OutputTimeUnit(TimeUnit.NANOSECONDS)
-  @BenchmarkMode(Array(Mode.AverageTime))
-  def fallthroughSwitch(input: BenchInput): Unit =
-    val inputStream = input.getInputStream
-    for i <- 0 until input.size do
-      val iri = RdfIriParseFallthrough.newInstance()
-      ProtoMessage.mergeDelimitedFrom(iri, inputStream)
-
-  @Benchmark
-  @OutputTimeUnit(TimeUnit.NANOSECONDS)
-  @BenchmarkMode(Array(Mode.AverageTime))
-  def tableSwitch(input: BenchInput): Unit =
-    val inputStream = input.getInputStream
-    for i <- 0 until input.size do
-      val iri = RdfIriParseTableswitch.newInstance()
-      ProtoMessage.mergeDelimitedFrom(iri, inputStream)
+//  @Benchmark
```

Comment:
commented out? should we remove it?

Reply:
I would keep it, this is dirty benchmarking code anyway, so it doesn't hurt.

I'm taking the approach of Jena where they just push JMH benchmarks in semi-working state to a non-published module.

Reply:
eh, ok, removed it, whatever